lib/reporters/JsonReporter.js

```diff
@@ -0,0 +1,34 @@
+const weblog = require('webpack-log');
+
+const WebpackCommandError = require('../WebpackCommandError');
+
+const Reporter = require('./Reporter');
+
+module.exports = class JesonReporter extends Reporter {
```

Comment:
Should this perhaps be `JsonReporter` instead of `JesonReporter`?

Reply:
jeez, yeah it should. I shouldn't PR so early in the morning 😆

Reply:
I kinda liked `JesonReporter` :smile: